testing/yarn/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=yarn
+pkgver=0.19.1
+pkgrel=0
+pkgdesc="Fast, reliable, and secure dependency management for Node.js."
+url="https://yarnpkg.com/"
+arch="noarch"
+license="BSD-2-Clause"
+depends="nodejs"
+source="https://github.com/yarnpkg/yarn/releases/download/v$pkgver/yarn-v$pkgver.tar.gz
+yarn.patch"
+builddir="$srcdir"/dist
+
+prepare() {
+	default_prepare || return 1
+	rm -rf "$srcdir"/dist/node_modules/node-uuid/benchmark/bench.gnu
+	find "$srcdir"/dist \( -name '*.md' -o  -name '*.md~' -o -name '*.gitmodules' \) -delete
+}
+
+package() {
+	mkdir -p "$pkgdir"/usr/share/yarn
+	mkdir -p "$pkgdir"/usr/bin
+
+	mv "$srcdir"/dist/bin "$pkgdir"/usr/share/yarn
+	mv "$srcdir"/dist/lib "$pkgdir"/usr/share/yarn
+	mv "$srcdir"/dist/lib-legacy "$pkgdir"/usr/share/yarn
+	mv "$srcdir"/dist/node_modules "$pkgdir"/usr/share/yarn
+	mv "$srcdir"/dist/package.json "$pkgdir"/usr/share/yarn
```

Comment:
We normally copy, or link things, so abuild can be ran multiple times for same source tree.